### ISSUE TITLE
Unexpected formatting with template and Japanese keyboard layout

### ISSUE THREAD

**Issue by Suhoy95** [elapouya/python-docx-template#255]
Template:
![Screenshot_1](https://user-images.githubusercontent.com/6908059/74173571-c6d00080-4c4b-11ea-8f3f-afaa76746c06.png)

Result:
![Screenshot_2](https://user-images.githubusercontent.com/6908059/74173575-c8012d80-4c4b-11ea-8ef3-4ec5b056e28a.png)

Formatting of Russian string is corrupted. Font "MS Mincho" is used (Why?).
Word 2007 is used. docxtpl version is `0.6.3`.

I tried [this](https://docxtpl.readthedocs.io/en/latest/#escaping-newline-new-paragraph-listing). 
- RichText: error of content "undeclared prefix", `part /word/header4.xml, line 1, column 23839` (translation from Russian)
![Screenshot_3](https://user-images.githubusercontent.com/6908059/74174779-03046080-4c4e-11ea-8f66-7bf8a6249e20.png)
I have tried to grep `upper case` function, but did not find why this happens.
- `|e` and `escape()` does not help

Any ideas how to workaround or fix it?

**Comment by Suhoy95** [comment]
Hm, it seems that I have to replace font manually at the end (the worst scenario)

**Comment by elapouya** [comment]
May be you could use RichText and specify an existing style in your
template.

Le lun. 10 févr. 2020 à 19:01, Илья Сухоплюев <notifications@github.com> a
écrit :

**Comment by Suhoy95** [comment]
Thanks for fast reply!

A little bit better, but still Word considers it as invalid because of uppercase: 
![Screenshot_4](https://user-images.githubusercontent.com/6908059/74179097-33e89380-4c56-11ea-8ffb-15aa407c58c2.png)

I have noticed that this behavior looks like a fallback: `MS Mincho` font is only in the `/word/fontTable.xml`. And there is some strange `w:eastAsia="ja-JP"` attribute.

I have solved it! I write ASCII template in Japan keyboard layout (it combines English, Hiragana and Katakana). So word template insert unexpected invisible formatting hint, which causes the problem. 

But RichText with uppercase is still kind of problem, but it is not necessary for me right now, I can continue to do my work.
